lnrpc/routerrpc/router.proto

```diff
@@ -61,6 +61,15 @@ service Router {
     rpc QueryMissionControl (QueryMissionControlRequest)
         returns (QueryMissionControlResponse);
 
+    /*
+    ImportMissionControl imports the state provided to the internal mission
+    control's state, using all results which are more recent than our existing
+    values. These values will only be imported in-memory, and will not be
+    persisted across restarts.
+    */
+    rpc ImportMissionControl (ImportMissionControlRequest)
```

Comment:
nit: still need to add the `X` prefix here as well as the Request/Response. that way if we decide to move it out of experimental (or if a different design ultimately emerges) there won't be any naming collisions for the desired names